### ISSUE TITLE
Manage Firefox dark mode

### ISSUE THREAD

**Issue by rubenvar** [rubenvar/csv-reader#9]
Firefox auto detects dark mode preference in the browser and adds:

```css
:root {
  background: #2a2a2e;
  color: rgb(249, 249, 250);
}
```

This messes up the table styling, because I don't set default styles for these values.

- Quick fix: hard code styles that overwrite the defult ones in `:root`:
  ```css
  :root {
    background: #fff;
    color: #000;
  }
  ```
- Better fix: add a dark mode to the extension (I'm already adding some dark styles when _rainbow-coloring_), and auto-detect user config.

**Comment by rubenvar** [comment]
Fixed in v1.1.2

**Comment by AGI-chandler** [comment]
It seems to be broken with TSV (tab separated values) file?  When such a file is displayed raw, my dark theme is shown with dark background and light text.  Example from https://gitlab.com/bu_cnio/bollito/-/raw/master/units-example_matrices_standard.tsv:
| ![](https://github.com/rubenvar/csv-reader/assets/27978373/955e559b-590e-4dc9-a60c-37466338ae78) |
|:-:|
| *Not activated* |

When activate CSV-reader "Try anyway" and use "\t" as a separator, "Convert" works and the formatting is nice, but is shown with bright white background and black text:
| ![image](https://github.com/rubenvar/csv-reader/assets/27978373/ab267e19-1746-4d89-af45-36a8c48d17ba) |
|:-:|
| *Convert* |

If i select the "Color code" button instead then it shows the data with nice colors and dark background, but the formatting is ruined:
| ![image](https://github.com/rubenvar/csv-reader/assets/27978373/7b51d506-1429-4f98-8cf3-a1b53cf6b990) |
|:-:|
| *Color code* |
